simpledotorg/simple-server
# Use new followup definition to check if a patient is lftu

**Story card:** [sc-3842](https://app.shortcut.com/simpledotorg/story/3824/change-patient-line-list-csvs-to-track-visits)

## Because

`lftu?` function  in the `materialized_patient_summary` was using an older follow-up definition

## This addresses

`ltfu` function now includes checks for if
- lastest blood pressure date < ltfu time 
- latest blood sugar date < ltfu time
- latest prescription drugs date < ltfu time
- latest appointment scheduled date < ltfu time

## Test instructions

Download Patient Line List CSV and verify that "patient status" column for each patient is marked correctly
